src/main/antlr/bio/terra/grammar/SQL.g4

```diff
@@ -94,13 +104,15 @@ alias_name  : name;
 array_name : name;
 column_name : name;
 cte_name : name;
+project_name : name;
 dataset_name : name;
 datatype_name : name;
 function_name : name;
 join_name : name;
 member_name : name;
 struct_name : name;
 table_name : name;
+extended_table_expr : (project_name '.')? table_expr;
```

Comment:
I just want to make sure that this is not a security problem (e.g. we should make sure that this doesn't allow a user to query a datasets table that they don't have access to)

Reply:
If this is a concern, then I'll just revert this change. It was just a drive-by.